Tests/SwiftFusionTests/Core/FixedShapeTensorTests.swift

```diff
@@ -47,12 +47,11 @@ class FixedShapeTensorTests: XCTestCase {
       }
     }
   }
-}
 
-class FixedShapeTensorVectorTests: XCTestCase, FixedSizeVectorTests {
-  typealias Testee = Tensor10x10
-  static var dimension: Int { 100 }
-  func testAll() {
-    runAllFixedSizeVectorTests()
+  func testVectorConformance() {
```

Comment:
I'm having trouble reviewing this change; the old and new content don't have an obvious correspondence.  Maybe you could explain?

Reply:
`runAllFixedSizeVectorTests` is what ran the old equivalent of the vector semantics checks. I have completely replaced it with the new vector semantics checks, which test pretty much the same thing but are written very differently.